test/integration/helpers/Env.cs

```diff
@@ -19,8 +19,8 @@ private static void Init()
 
             _env = new Dictionary<string, JsonElement>
             {
-                { "DEV", JsonDocument.Parse("true").RootElement }, 
-                { "isRemoteAppiumServer", JsonDocument.Parse("false").RootElement }, 
+                { "DEV", JsonDocument.Parse("true").RootElement },
+                { "isRemoteAppiumServer", JsonDocument.Parse("false").RootElement },
```

Comment:
What's the change here?

Reply:
Hm, let me add spaces. My VSCode trims spaces at the end automatically.